Test Host/PickerController.m

```diff
@@ -99,50 +115,100 @@ - (void)viewDidLoad
     countdownSelectionTextField.returnKeyType = UIReturnKeyDone;
     countdownSelectionTextField.inputView = countdownPicker;
     countdownSelectionTextField.accessibilityLabel = @"Countdown Selection";
+    
+    self.dateCalendarPicker = [[UIDatePicker alloc] init];
```

Comment:
I assume these are using `preferredDatePickerStyle` of automatic? Should we be more explicit here and choose `UIDatePickerStyleCompact`? Should we be handling `UIDatePickerStyleInline` as well, or considering that out of scope for now?

Reply:
Also, should these fields only be exposed and tested for iOS 14+? Looks like compact mode is supported in 13.4+ from the docs.

Reply:
Yep i have them in 13.4+. Inline isn't any different of styles just is that is already tested so i was thinking we can just  test the compact and wheel types